eecs485staff/primer-spec
# [Hotfix] Un-garble print previews in Chrome

## Context

This PR introduces a work-around for garbled print-previews in Google Chrome: If the user's browser is Chrome, and version < 82, then remove all italicized formatting on the page before printing. The changes that enable this functionality are in 498699d63dba408ed0ffba29fda85e1c2fb3d609.

This PR also bumps the current Primer Spec version. When this PR merges with `master`, all specs that use Primer Spec v.1.1.0 will automatically be upgraded to v.1.1.1. (No further action is needed by EECS 485 / EECS 280 staff.)

## Validation

- Visit https://eecs485staff.github.io/primer-spec/ with Google Chrome, then try generating a print preview. The preview should appear garbled.

- Then visit the [preview URL for this PR](https://preview.seshrs.ml/previews/eecs485staff/primer-spec/40/) on Google Chrome, and try generating a print preview — it should look all right (with italics formatting removed).

- Visit the preview URL from a non-Chromium browser (like Safari or Firefox). Print previews should be generated correctly, and should include italicized formatting.

/CC @rileyschnee

Closes #38, and closes #39.
